celements-model/src/main/java/com/celements/rights/access/RightsAccessScriptService.java

```diff
@@ -75,4 +94,15 @@ public boolean isSuperAdmin() {
   public boolean isLayoutEditor() {
     return rightsAccess.isLayoutEditor();
   }
+
+  public List<DocumentReference> getGroupRefsForUser(DocumentReference userDocRef) {
+    try {
+      User user = userService.getUser(userDocRef);
```

Comment:
getUser parameter not nullable, but ScriptService paramters should always be nullable